code/game/objects/items/stacks/stack.dm

```diff
@@ -132,6 +132,14 @@
 	else
 		icon_state = "[initial(icon_state)]_3"
 
+/obj/item/stack/update_overlays()
+	. = ..()
+	if(isturf(loc))
+		return
```

Comment:
Doesn't ore stacks make use of overlays for their own stack system? Wouldn't this interfere with that?

Reply:
This is a legitimate issue but also storage code sucks, i'm stumped and i hate it.

Reply:
This should probably just be `return .`

From there that should Just Work

```suggestion
		return .
```

Reply:
Not actually a problem, `return` always returns what's on `.`